pkg/api/team.go

```diff
@@ -102,6 +104,20 @@ func (hs *HTTPServer) DeleteTeamByID(c *models.ReqContext) response.Response {
 	return response.Success("Team deleted")
 }
 
+func (hs *HTTPServer) getTeamsAccessControlMetadata(c *models.ReqContext, teamIDs map[string]bool) (map[string]accesscontrol.Metadata, error) {
+	if hs.AccessControl.IsDisabled() || !c.QueryBool("accesscontrol") {
```

Comment:
I have not seen this `c.QueryBool("access control")`

what is the difference between the IsDisabled() vs this one 🤔

Reply:
This boolean is a query param, we only return metadata when it's appended to the URL, ex: `/api/teams/:teamId?accesscontrol=true`